filters/tracing/tag.go

```diff
@@ -46,23 +81,60 @@ func (s *tagSpec) CreateFilter(args []interface{}) (filters.Filter, error) {
 		return nil, filters.ErrInvalidFilterParameters
 	}
 
-	return &tagFilter{
-		tagFromResponse: s.typ == filters.TracingTagFromResponseName,
-
+	f := &tagFilter{
+		typ:      typ,
 		tagName:  tagName,
 		tagValue: eskip.NewTemplate(tagValue),
-	}, nil
+	}
+
+	if len(args) == 3 {
+		conditionString, ok := args[2].(string)
+		if !ok || len(conditionString) < 2 {
+			return nil, filters.ErrInvalidFilterParameters
+		}
+		condValueStr := conditionString[1:]
+		condValue, err := strconv.Atoi(condValueStr)
+		if err != nil {
+			return nil, filters.ErrInvalidFilterParameters
+		}
+
+		b := conditionString[0]
+		switch b {
+		case '<':
+			f.condition = func(rsp *http.Response) bool {
+				return rsp.StatusCode < condValue
+			}
+		case '>':
+			f.condition = func(rsp *http.Response) bool {
+				return rsp.StatusCode > condValue
+			}
+		case '=':
+			f.condition = func(rsp *http.Response) bool {
+				return rsp.StatusCode == condValue
+			}
+		default:
+			return nil, filters.ErrInvalidFilterParameters
+		}
```

Comment:
Both `tracingTag` and `tracingTagFromResponse` currently receive two parameters: tag name and value. We could add a third optional parameter that is a [Common Expression Language](https://github.com/google/cel-go) expression like:
```
tracingTagFromResponse("foo", "bar", "response.StatusCode >= 500 && response.StatusCode <= 599")
```

Reply:
as discussed: CEL is future vision and we collect ideas